tests/cardio_test.cpp

```diff
@@ -274,24 +274,6 @@ TEST_CASE( "cardio_is_affected_by_activity_level_each_day", "[cardio][activity]"
     // Then their cardio should increase slightly
 }
 
-TEST_CASE( "cardio_is_not_affected_by_character_height", "[cardio][height]" )
```

Comment:
Your PR has no explanation for why this test ended up being fully removed, as opposed to just modified.